tests/app_tests/endpoint_tests/posts_test.go

```diff
@@ -40,3 +40,40 @@ func TestPostSuccess(t *testing.T) {
 	assert.Equal(t, http.StatusOK, w.Code)
 	assert.Contains(t, w.Body.String(), "TestPost")
 }
+
+func TestPostFailure(t *testing.T) {
+
+	app_settings := common.AppSettings{
+		DatabaseAddress:  "localhost",
+		DatabasePort:     3006,
+		DatabaseUser:     "root",
+		DatabasePassword: "root",
+		DatabaseName:     "urchin",
+		WebserverPort:    8080,
+	}
+
+	database_mock := mocks.DatabaseMock{
```

Comment:
You can just do `mocks.DatabaseMock{}` here, the post handler won't get called in the code.